File: UserDomain/UpdateUserCoverService/routes/updateUserCover.js
var express = require('express');
var historiasController = require('../controllers/updateUserCoverController');
var auth = require('../middlewares/auth');
var multiparty = require('connect-multiparty');
var path = multiparty({uploadDir:'./uploads/stories'});


var app = express.Router();

app.post('/update_user_cover',[auth.auth,path],updateUserCoverController.update_user_cover);

module.exports = app;

File: UserDomain/UpdateUserCoverService/routes/update_user_cover.py
from fastapi import APIRouter, Depends, File, UploadFile
from controller.update_user_cover_controller import update_user_cover
from middlewares.auth import auth

router = APIRouter()

@router.post("/update_user_cover")
async def update_cover(file: UploadFile = File(...), user_data=Depends(auth)):
    return await update_user_cover(user_data, file)
